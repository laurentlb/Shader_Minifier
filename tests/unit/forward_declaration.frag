void a();
void b();
void c();
void d();

void a() {
  b();
}

void b() {
  c();
  d();
}

void e() {
  a();
  c();
}

int x;
void c() {x;}

void d() {x;}
