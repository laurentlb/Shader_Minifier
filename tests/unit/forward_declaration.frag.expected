int x;
void c()
{
  x++;
}
void d()
{
  x++;
}
void b()
{
  c();
  d();
}
void a()
{
  b();
}
void e()
{
  a();
  c();
}
